app/screens/post_options/index.js

```diff
@@ -39,6 +41,7 @@ function mapStateToProps(state, ownProps) {
     const channelIsArchived = channel.delete_at !== 0;
 
     let canAddReaction = true;
+    let canReply = true;
```

Comment:
What if you are already in the thread view? this option should not be included if that is the case

Reply:
Good catch!